build/charts/yorkie-mongodb/values.yaml

```diff
@@ -0,0 +1,189 @@
+namespace: &namespace mongodb
+name: &name mongodb
+port: &port 27017
+clusterDomain: &clusterDomain cluster.local
+
+image:
```

Comment:
Could you add descriptions for each section(image, persistence, sharded, and mongodb-sharded)?